spec/factories/users.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :user do
+    sequence(:email) { |n| "user_#{n}@example.com" }
+    password { "password" }
+    password_confirmation { "password" }
```

Comment:
同上ですが、シングルクォートで囲っておきましょう。
```suggestion
    password { 'password' }
    password_confirmation { 'password' }
```